CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Fixed
 
+- Fix bug where the console was enabled by
+ `OTEL_DOTNET_AUTO_{SIGNAL}_CONSOLE_EXPORTER_ENABLED` and wasn't working
+
```

Comment:
This entry is not needed. You are fixing issue introduced in the very same release :- )